src/column.rs

```diff
@@ -886,6 +887,7 @@ impl Column {
 		change: &Operation<K, V>,
 		log: &mut LogWriter,
 		stats: Option<&ColumnStats>,
+		btree_index_node: bool,
```

Comment:
Maybe just call this `ref_counted` and simply check `if ref_counted` instead of `if tables.ref_counted && !btree_index_node`?

Reply:
:facepalm: clearly yes.